TensorFlow/scripts/annotation_parser.py

```diff
@@ -0,0 +1,41 @@
+from xml.dom import minidom
+
+
+class AnnotationsParser:
```

Comment:
Why did you copy this script? Please add changes to previously created one and use it.

Reply:
Ok, I'll fix that myself today